src/rpi/minimal.cpp

```diff
@@ -169,9 +169,9 @@ int init_SDL(void)
 				//Allow SDL badly handled axis like PS3
 				if (SDL_JoystickNumAxes(myjoy[i]) > 28)
 				{
-					SDL_JoystickClose(myjoy[i]);
-					myjoy[i]=0;
-					logerror("Error detected invalid joystick/keyboard\n");
+					logerror("Error detected invalid joystick/keyboard. Detected %d axis\n", SDL_JoystickNumAxes(myjoy[i]));
+					//SDL_JoystickClose(myjoy[i]);
+					//myjoy[i]=0;
 				}
 			}
 		}
```

Comment:
It was hard to say.  It seems like it's there to prevent issues with certain other devices, but it's very apparent now that Wireless PS3 is still an issue.  I'm fine with removing it all together.  Just hoping that someone in the future doesn't put it back for other reasons.